AppleSignInAuthentication/README.md

```diff
@@ -0,0 +1,51 @@
+# Integrating Apple Sign-in with Realm using the Swift SDK: A Step-by-Step Guide
+
+This example contains a skeleton app to be used as a reference on how to implement Apple Sign in authentication with Realm using the [Realm Swift SDK](https://www.mongodb.com/docs/atlas/app-services/authentication/apple/#apple-id-authentication).
+In this project you will be able to see an example of the following use case
+* Use Apple Sign In implementation in order to authenticate users in your Realm database.
+	* Log screen view in UIKit.
```

Comment:
Are these subitems of the above point? Should

Reply:
They are contained as a section inside the main headline, yes